modules/installation-initializing.adoc

```diff
@@ -535,6 +536,7 @@ ifeval::["{context}" == "installing-gcp-vpc"]
 endif::[]
 ifeval::["{context}" == "installing-gcp-user-infra"]
 :!gcp:
+:!azure-user-infra:
```

Comment:
This unset should be happening up under ~line 525 instead:

```
ifeval::["{context}" == "installing-azure-user-infra"]
```

Since that's the context under which you had initially set the new attribute.

Reply:
Incorporated